src/shrink.rs

```diff
@@ -303,10 +303,12 @@ impl Shrinker for StdShrinker {
             &mut StdShrinkerBody::Div8(ref mut shrinker) =>
```

Comment:
I guess that's a fix for another time, but this would be in more usual style if all the `&mut` were dropped.